src/viur/core/modules/user.py

```diff
@@ -830,9 +996,9 @@ class User(List):
     passwordRecoveryMail = "user_password_recovery"
 
     authenticationProviders = [UserPassword, GoogleAccount]
-    secondFactorProviders = [TimeBasedOTP]
-
-    validAuthenticationMethods = [(UserPassword, TimeBasedOTP), (UserPassword, None), (GoogleAccount, None)]
+    secondFactorProviders = [TimeBasedOTP, AuthenticatorOTP]
+    validAuthenticationMethods = [(UserPassword, AuthenticatorOTP), (UserPassword, TimeBasedOTP), (UserPassword, None),
+                                  (GoogleAccount, None)]
```

Comment:
Make code better readable and maintainable here.
```suggestion
    validAuthenticationMethods = [
        (UserPassword, AuthenticatorOTP),
        (UserPassword, TimeBasedOTP),
        (UserPassword, None),
        (GoogleAccount, None),
    ]
```